lua/fishing_mod/cl_init.lua

```diff
@@ -67,17 +67,17 @@ end)
 hook.Add("Think", "Fishingmod.Keys:Think", function()
 	local ply = LocalPlayer()
 	if ply:GetFishingRod() and not vgui.CursorVisible() then
-		if input.IsKeyDown(KEY_B) then
+		if input.IsKeyDown(KEY_B) and not input.LookupKeyBinding(KEY_B) or input.LookupKeyBinding(KEY_B) == "" and input.IsKeyDown(KEY_B) then -- only open menu when the player has nothing bound to the key

```

Comment:
```suggestion
		if input.IsKeyDown(KEY_B) and (not input.LookupKeyBinding(KEY_B) or input.LookupKeyBinding(KEY_B) == "") then -- only open menu when the player has nothing bound to the key

```
You can use `()` here to avoid calling `input.IsKeyDown(KEY_B)` again and avoid redundancy.